examples/memoize-simple.yaml

```diff
@@ -22,7 +22,7 @@ spec:
   arguments:
     parameters:
```

Comment:
can you delete this?

Reply:
Done. This could simplify the example a bit.